src/Offer/ReadModel/JSONLD/SameAsOfferRepositoryDecorator.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Offer\ReadModel\JSONLD;
+
+use CultuurNet\UDB3\ReadModel\DocumentRepository;
+use CultuurNet\UDB3\ReadModel\DocumentRepositoryDecorator;
+use CultuurNet\UDB3\ReadModel\JsonDocument;
+use CultuurNet\UDB3\SameAsInterface;
+
+final class SameAsOfferRepositoryDecorator extends DocumentRepositoryDecorator
+{
+    private SameAsInterface $sameAs;
+
+    public function __construct(DocumentRepository $documentRepository, SameAsInterface $sameAs)
+    {
+        parent::__construct($documentRepository);
+        $this->sameAs = $sameAs;
```

Comment:
Feel free to inline the construction of the sameAsGenerator, no need to inject it.

Reply:
Since I've moved it to NewPropertyPolyfillOfferRepository it's no longer relevant.